setup.cfg

```diff
@@ -111,7 +112,6 @@ testing =
     fsspec
     zarr
     matplotlib
-    scikit-image>=0.18.1,!=0.19.0
```

Comment:
Probably there needs to be `triangle` added to a section... Maybe a new section [optional] or [performance]?

Reply:
we could do that, if as an *optional* dep we circumvent the [license issue above](https://github.com/napari/napari/pull/3867#issuecomment-999978209)? @sofroniewn

Reply:
i believe these are just "testing" dependencies and so as we don't ship with them we don't have to worry about the license issue here

Reply:
Yes, but its not really related to `testing` per se. (btw, `pip install[testing]` seems to fail on M1 anyhow to `tensorstore` not being able to be built)

Reply:
yup yup, sorry - i guess maybe doing `[performance]` is fine. i think the optional aspect of it makes it ok for licensing. if we make it my default in the bundle then we might need to add something special, but that can come later

Reply:
Would `all` be too much?

Reply:
`all` means we'll hit that license issue for sure - not sure if we're ready to commit to that

Reply:
Yeah, I agree. Might be its best to avoid specificity `triangle` as a dependency at all and let users simply install if for themselves? ([this seems to be the way vispy does it](https://vispy.org/installation.html#optional-dependencies))

Reply:
yeah, let's just hold off for now - we can do something for next release

Reply:
@sofroniewn @maweigert PySide2 is LGPL2 anyway so I don't understand what we are worried about here. And note that "all" currently depends on PyQt5 which is GPL!

Reply:
@sofroniewn @jni Ok, I feel like getting opposing gradients here ;) 

I think immediate options for now are

A) Don't add any dep and let user `pip install triangle`
B) Add it to `all`, thus effectively making the dep opt-out  (as most people will probably do `pip install napari[all]` anyway)
C) Create a separate dep section that inherits from `all` while adding even more opt-in deps such as `triangle` (might be thus called `uberall`)

Reply:
I think I like (D) add [performance]. We can later include it or not include it in [all], but for now it makes it easier to say `pip install napari[all,performance]`. (I do think it should go in [all] for the record, but that will slow down discussion.)

And yes you're getting opposing gradients but imho mine is the correct gradient, @sofroniewn's is just noise. :stuck_out_tongue_winking_eye: :joy: More seriously, I really don't see how this in any way affects our licensing, since we already depend on an LGPL package even for the bundle, and a GPL package in [all].

Reply:
Hahah, ok :) 

So I opted for (D'), which is your suggestion of adding a new section, but I named it `[optional]` (which might make it more transparent).